azure/microservice-ecommerce/xebialabs/xlr-pipeline-ci-cd.yaml.tmpl

```diff
@@ -86,10 +86,19 @@ spec:
       tasks:
       {{if .ProvisionCluster }}
       - name: Deploy {{.Namespace}} namespace
-        type: xldeploy.Deploy
-        server: XL Deploy
-        deploymentPackage: {{$app}}/K8S/{{$app}}-namespace/1.0.0
-        deploymentEnvironment: Environments/{{$app}}/{{$appEnv}}
+        type: xlrelease.SequentialGroup
+        tasks:
+        - name: Deploy namespace
+          type: xldeploy.Deploy
+          server: XL Deploy
+          deploymentPackage: {{$app}}/K8S/{{$app}}-namespace/1.0.0
+          deploymentEnvironment: Environments/{{$app}}/{{$appEnv}}
+        - name: Tag namespace
+          type: xld.UpdateCIProperty
+          server: XL Deploy
+          ciID: Infrastructure/{{$app}}/{{$app}}-AKSCluster/{{.Namespace}}
```

Comment:
This targets `Infrastructure/{{$app}}/{{$app}}-AKSCluster/{{.Namespace}}`, but should probably target `Infrastructure/{{$app}}/{{$clusterName}}-AKSCluster/{{.Namespace}}`

Reply:
done